devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy

```diff
@@ -108,6 +108,8 @@ metadata {
 		fingerprint manufacturer: "Jasco Products", model: "43094", deviceJoinName: "Enbrighten Switch" //Enbrighten, Plug-in Smart Switch 43094, Raw Description: 01 0104 0100 00 06 0000 0003 0004 0005 0006 0B05 02 000A 0019
 		fingerprint manufacturer: "Jasco Products", model: "43102", deviceJoinName: "Enbrighten Outlet", ocfDeviceType: "oic.d.smartplug" //Enbrighten, In-Wall Smart Outlet 43102, Raw Description: 01 0104 0100 00 06 0000 0003 0004 0005 0006 0B05 02 000A 0019
 		fingerprint manufacturer: "Jasco Products", model: "43076", deviceJoinName: "Enbrighten Switch" //Enbrighten, In-Wall Smart Switch 43076, Raw Description: 01 0104 0100 00 06 0000 0003 0004 0005 0006 0B05 02 000A 0019
+        // Focalcrest
+        fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0019", outClusters: "0019", manufacturer: "Focalcrest", model: "SRB01", deviceJoinName: "In-Wall Relay Switch" // In-Wall Relay Switch
```

Comment:
Please change deviceJoinName to "Focalcrest Switch" and fix spacing (use tabs instead of spaces / 4 spaces -> 1 tab)

Reply:
@KKlimczukS 
I have modified deviceJoinName and fixed spacing
Why can't the name use "In-Wall Relay Switch"？
Why use "Focalcrest Switch"？

Reply:
@MadiTang That's because of internal deviceJoinName naming rules policy.